Fails to spawn relative modules in child processes when used with `process.execPath`

Source files:

```
$ tree
.
├── sub
│   └── c.js
└── t.js

1 directory, 2 files

$ cat sub/c.js
console.log(process.execPath);

$ cat t.js
const { spawn } = require('child_process');

spawn(
  process.execPath,
  ['c.js'],
  { cwd: '/Users/nrajlich/t/sub', stdio: 'inherit' }
);
```

When invoking this script directly with `node`, it works as expected:

```
$ node t
/usr/local/bin/node
```

Now let's try it with a `pkg` binary:

```
$ pkg t.js
> pkg@4.3.7
> Targets not specified. Assuming:
  node10-linux-x64, node10-macos-x64, node10-win-x64

$ ./t-macos
internal/modules/cjs/loader.js:594
    throw err;
    ^

Error: Cannot find module 'c.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:592:15)
    at Function.Module._resolveFilename (pkg/prelude/bootstrap.js:1280:46)
    at Function.Module._load (internal/modules/cjs/loader.js:518:25)
    at Function.Module.runMain (pkg/prelude/bootstrap.js:1309:12)
    at startup (internal/bootstrap/node.js:274:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:608:3)
```

I'm guessing that it's trying to look up `c.js` from within the snapshot filesystem because it's a relative path instead of an absolute one.

--- a/package.json
+++ b/package.json
@@ -171,7 +171,7 @@
     "nyc": "13.2.0",
     "ora": "1.3.0",
     "pcre-to-regexp": "0.0.5",
-    "pkg": "4.3.7",
+    "pkg": "4.3.8",
     "pluralize": "7.0.0",
     "pre-commit": "1.2.2",
     "prettier": "1.16.2",

